cfpb/api
# Redirect old HMDA API documentation URLs to the new docs

**Do not merge until Sept. 25**

The old HMDA API is being decommissioned, so we need to redirect the old docs to the new docs.

### Additions

- In what looks like an oversight, there was no page at the root, so http://cfpb.github.io/api/ currently shows an unstyled version of the repo's README. I added a simple `index.html` which just redirects to the CCDB API's landing page, since those docs will be the only thing left here once the HMDA API docs are removed.

### Removals

- All of the documentation content in the markdown files
- All of the `console` static files

### Changes

- All HMDA API github.io documentation pages now redirect to the new documentation at https://cfpb.github.io/hmda-platform/#data-browser-api. I did that redirect with a `<meta>` element since we can't do anything server side (we did [the same thing in the Design Manual](https://github.com/cfpb/design-manual/blob/gh-pages/index.html#L4) when we shut that down).

### Testing

1. Fire up this branch locally with `bundle exec jekyll serve --watch --baseurl '/api'`
2. Visit any HMDA API docs page, like `/api/hmda`, `/api/hmda/console`, or `/api/hmda/queries`; you should be immediately (or almost immediately) redirected to https://cfpb.github.io/hmda-platform/#data-browser-api
3. Visit the API landing page, `/api`; you should be redirected to the CCDB API docs landing page, `/api/ccdb`
4. Visit some CCDB API documentation pages (anything under `/api/ccdb/`) to verify that they are unchanged and still match what's live right now.
